.github/workflows/python_client_checks.yml

```diff
@@ -1,5 +1,9 @@
 name: Checks on the Python client
 
+# This workflow performs some checks on the Python client used by the Boilerplate tests
```

Comment:
Why not a reusable workflow for this?

Reply:
IMO it's very specific to this app, the point of this CI check is to ensure that the boilerplate example client is clean and ready to be forked. We don't plan to enforce such checks on the test clients of other apps